docs/dev/cli-commands.md

```diff
@@ -0,0 +1,38 @@
+## CLI Basics
+
+The main tasks of the CLI component are:
+- parse and call the commands
+- call the method to transform a given topology to a custom platform (start / stop / transform platform)
+- show current status of transformation/ deployment
+- print detailed logs if wanted
+- show supported platforms
+- create threads to provide asynchronous operations
+
+## Supported Commands
```

Comment:
do i just type my command or do I need to enter something like `toscana add ...`?
Please specify detailed how to use the cli.